examples/variational_autoencoder.py

```diff
@@ -39,7 +41,7 @@ def sampling(args):
 
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
+    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
IAF was an earlier reference. it uses conditional embedding but it is still fine I think. vae-gan is beyond the scope